docs/api/version-history.md

```diff
@@ -34,6 +34,9 @@ keywords: "API, Docker, rcli, REST, documentation"
   (`a disk usage operation is already running`) would be returned in this
   situation. This change is not versioned, and affects all API versions if the
   daemon has this patch.
+* Removed the `KernelMemory` field in `HostConfig` on the `POST /containers/create`
```

Comment:
🤦🤦🤦 So I'm reading this, and just realised I had to check if `docker container update` also supported this, and it looks like it does, so we need to update the `/containers/{id}/update` endpoint as well; https://github.com/moby/moby/blob/9e7bbdb9ba53440cfc67e2e1de89e35e44c43ddf/api/server/router/container/container_routes.go#L421-L437